### PR TITLE
Fix accessibility issues

### PR DESCRIPTION
**Pull request by saliceti**
- Fix accessibility issues
- Update govuk tech docs gem
- Update accessibility statement

### CODE REVIEW

**Comment by saliceti** [comment]
[Update tech guidance accessibility statement](https://trello.com/c/29Q3UazD/266-update-tech-guidance-accessibility-statement)

**Review by aje54 — APPROVED** [review]

File: source/accessibility.html.md.erb [hunk 1]
```diff
@@ -69,9 +57,3 @@ We used manual and automated tests to look for issues such as:
 - lack of colour contrast for text, important graphics and controls
 - images not having meaningful alt text
 - problems when using assistive technologies such as screen readers and screen magnifiers
-
-## What we’re doing to improve accessibility
-
-When we publish new content, we’ll make sure our use of content meets accessibility standards.
-
-This statement was prepared on 24 September 2020. It was last updated on 24 September 2020.
```
Comment (luke-slowen) [inline source/accessibility.html.md.erb:77]: I've a feeling the statement last updated date also needs to stay as a line of text, in addition to it being in the header.

File: source/accessibility.html.md.erb [hunk 2]
```diff
@@ -45,18 +43,8 @@ The Equality and Human Rights Commission (EHRC) is responsible for enforcing the
 
 DfE is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
 
-## Compliance status
```
Comment (luke-slowen) [inline source/accessibility.html.md.erb:48]: Compliance statement needs to stay - https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website

**Review by luke-slowen — COMMENTED** [review]
Can't comment on the Gemfile package updates, couple of things to resolve on the accessibility wording.

File: source/accessibility.html.md.erb (hunk 1 above)
Comment (saliceti) [reply source/accessibility.html.md.erb:77]: I removed it on purpose as I'm trying to simplify the whole statement and it's a burden to update the date is multiple places as they can easily be missed.

File: source/accessibility.html.md.erb (hunk 2 above)
Comment (saliceti) [reply source/accessibility.html.md.erb]: Thanks, I've added it back

**Pull request merged**